front/components/assistant/conversation/UserMessage.tsx

```diff
@@ -48,8 +48,8 @@ export function UserMessage({
   );
 
   return (
-    <div className="flex w-full flex-col">
-      <div className="flex flex-col items-end gap-6">
+    <div className="flex flex-grow flex-col">
```

Comment:
It works anyway so this is just a nitpick but seems like we don't need to have `flex-grow` here, the parent element of `UserMessage` is not flex item so this doesn't do anything 😄

``` 
          <div
            key={`message-id-${sId}`}
            ref={ref}
            className="min-w-60 max-w-full"
          >
            <UserMessage
              citations={citations}
              conversationId={conversationId}
              isLastMessage={isLastMessage}
              message={message}
              owner={owner}
            />
          </div>

```